### PR TITLE
Xml fixes for Magento_AdvancedPricingImportExport module

### PR DESCRIPTION
**Pull request by sanganinamrata**
### Description (*)

Finetuned module.xml

### Related Pull Requests

N/A

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios (*)

N/A

### CODE REVIEW

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-6784](https://jira.corp.magento.com/browse/ENGCOM-6784) has been created to process this Pull Request

File: app/code/Magento/AdvancedPricingImportExport/etc/module.xml [hunk 1]
```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_AdvancedPricingImportExport" >
-    </module>
+    <module name="Magento_AdvancedPricingImportExport" />
```
Comment (lbajsarowicz) [inline app/code/Magento/AdvancedPricingImportExport/etc/module.xml]: Remove space before `/>`

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Lint your code.

**Review by ihor-sviziev — APPROVED** [review]

File: app/code/Magento/AdvancedPricingImportExport/etc/module.xml (hunk 1 above)
Comment (ihor-sviziev) [reply app/code/Magento/AdvancedPricingImportExport/etc/module.xml]: Fixed by myself, it’s too small for waiting fix from contributor

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-6784](https://jira.corp.magento.com/browse/ENGCOM-6784) has been created to process this Pull Request

**Review by lbajsarowicz — APPROVED** [review]
OK.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6784](https://jira.corp.magento.com/browse/ENGCOM-6784) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Charlie** [comment]
Failure of functional test, not related to the changes in this PR , test failing on 2.4-develop branch
73529028-a23f8180-441e-11ea-8642-6d0f12b8b6de

**Pull request merged**